query-node/schemas/validator.graphql

```diff
@@ -0,0 +1,6 @@
+type Validator @entity {
+  id: ID!
+  isValidator: Boolean!
+  validatorAccount: String
+  member: String
```

Comment:
The member field should be of type `Membership`.

The way the QN are structured, `Validator` should be available on `MembershipMetadata` because this entity linking a validator to a membership only exist in the QN and not really on chain. For this reason maybe `Validator` should be renamed `ValidatorProfile`. That said due to some hydra limitation we have to also link this entity to the `Membership` entity in order to be able to filter members by validator addresses and state. So this should look like this:
```suggestion
  memberMetadata: MemberMetadata!
  member: Membership!
```